README.md

```diff
@@ -114,6 +114,20 @@ If needed, especially for low memory situations the memory limit for all adapter
 }
 ```
 
+### Directly exeuting Typescript adapters
+**Feature status:** Technology preview since js-controller 3.3.0
+
+The js-controller is able to execute `.ts` files, which removes the need of compiling to Javascript first.
+To use this feature, simply define the adapter main file as the required `.ts` file.
```

Comment:
Additional points to consider:
- The `src` directory needs to be released as part of the npm package (--> .npmignore / `package.json files`)
- The `build` directory probably shouldn't be created (disable compilation scripts/automations) or at least not checked in to GitHub / released to `npm`.
- The adapter is compiled at every startup (with the very performant ESBuild, but that still takes some additional resources)

Reply:
Quesiton is do we want to document this here?

Reply:
Maybe only the last point as a thing to consider. The others are more suited for the dev docs.